static/app/views/releases/components/header.tsx

```diff
@@ -63,6 +72,25 @@ function Header({router, hasV2ReleaseUIEnabled}: Props) {
           />
         </Layout.Title>
       </Layout.HeaderContent>
+      {hasV2ReleaseUIEnabled && selected === THRESHOLDS_PATH && (
+        <Layout.HeaderActions>
+          <Tooltip
+            title="Select a single project from the project filters to create a threshold for"
```

Comment:
Product question, can you select multiple projects? The selector is multi-select and selecting just one makes it confusing.

Edit: I saw lower down in the code that it's actually multi select. Where does this tooltip show up?

Reply:
Yup - this tooltip shows up on the "create threshold" button.
I added a video to the description so you can see it there